cfme/common/vm.py

```diff
@@ -438,6 +438,38 @@ def unset_ownership(self):
         view = self.create_view(navigator.get_class(self, 'Details').VIEW)
         view.flash.assert_success_message(msg)
 
+    def rename_vm(self, new_vm_name, old_vm_name, click_cancel=False, click_reset=False):
+        """Rename the VM
+
+        Args:
+            new_vm_name: object for renaming vm
+            old_vm_name: object for assertion
+            click_cancel (bool): Whether to cancel form submission
+            click_reset (bool): Whether to reset form after filling
+        """
+        view = navigate_to(self, 'RenameVM', wait_for_view=0)
+        view.form.fill({'vm_name': new_vm_name})
+        if click_reset:
+            view.form.reset_button.click()
+            view.flash.assert_message('All changes have been reset', 'warning')
+            # Cancel after reset
+            assert view.form.is_displayed
+            view.form.cancel_button.click()
+
+        elif click_cancel:
+            view.form.cancel_button.click()
+            view.flash.assert_message('Rename of VM "{}" was cancelled by the user'
+                                      .format(old_vm_name))
+
+        else:
+            # save the form
+            view.form.save_button.click()
+            view = self.create_view(navigator.get_class(self, 'Details').VIEW)
+            view.wait_displayed("60s")
+            view.flash.assert_success_message('Rename of Virtual Machine "{}" has been initiated'
```

Comment:
Required:
Please avoid asserting such messages in the `model`. you can assert this message in test